Sources/AppCoinsSDK/Models/Services/Billing/AppCoinBillingClient.swift

```diff
@@ -15,18 +15,13 @@ internal class AppCoinBillingClient : AppCoinBillingService {
         self.endpoint = endpoint
     }
 
-    internal func convertCurrency(money: String, fromCurrency: Coin, toCurrency: Coin?, result: @escaping (Result<ConvertCurrencyRaw, BillingError>) -> Void) {
+    internal func convertCurrency(money: String, fromCurrency: String, toCurrency: String?, result: @escaping (Result<ConvertCurrencyRaw, BillingError>) -> Void) {
```

Comment:
If we now have default currencies such as APPC on the Currency class maybe we can have Currency instead of String here, no?